x-pack/plugins/search_playground/public/plugin.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CoreSetup, Plugin, CoreStart, AppMountParameters } from '@kbn/core/public';
+import { PLUGIN_ID, PLUGIN_NAME } from '../common';
+import { PlaygroundToolbar, PlaygroundProvider, Playground } from './embedable';
+import {
+  AppPluginStartDependencies,
+  SearchPlaygroundPluginSetup,
+  SearchPlaygroundPluginStart,
+} from './types';
+
+export class SearchPlaygroundPlugin
+  implements Plugin<SearchPlaygroundPluginSetup, SearchPlaygroundPluginStart>
+{
+  public setup(core: CoreSetup): SearchPlaygroundPluginSetup {
+    return {};
```

Comment:
This should have a config value to gate this, we can punt that to when we want to support serverless though.

Reply:
Yes lets punt this to the serverless ticket.